README.md

```diff
@@ -42,10 +46,27 @@ Usage of ./server:
     	print version number and build date
 ```
 
-If you are working on the sqlite migrations, html templates or website assets, build the server with `go build -tags dev`.
-This way it won't use the assets that are embedded in the binary but read them directly from the local filesystem.
+If you want to view the development server in your browser:
+```sh
+# change to the root of the project (e.g. cd go-ssb-room) and generate the frontend's styling; requires npm
+go generate -tags dev ./...
+# now let's build & run the development server
+cd cmd/server && go build -tags dev && ./server
+# and visit http://localhost:3000
+```
+
+This can be useful if you are working on:
+* the sqlite migrations, 
+* html templates, 
+* or website assets
 
-Once you are done with your changes run `go generate` in the changed packages to update them.
+This way, the build won't use the assets embedded in the binary, but instead read them directly from the local filesystem.
+
+Once you are done with your changes and want to update the embedded assets:
```

Comment:
This updating is no longer required since the embedding happens at build time via the Go1.16 [embed](https://golang.org/pkg/embed/) package.

The previous setup generated `_vfsgen.go` files, which included the contents of the assets/templates folder but they are all gone now, which means, no updating required :)

Reply:
ya i got that previously, but still a bit confused by the tailwind situation. so they are not being embedded? or are they?